### ISSUE TITLE
Tab separators disappear when tabs are moved

### ISSUE THREAD

**Issue by dreamsynth** [QNetITQ/WaveFox#189]
Even though the tab separators is enabled, they disappear when tabs are moved.

![изображение](https://github.com/QNetITQ/WaveFox/assets/84819091/80520b24-2380-414f-b02e-981926a21276)

Firefox Dev 123.0b7
`userChrome.Tabs.Option2.Enabled`
`userChrome.TabSeparators.Saturation.Low.Enabled`
`userChrome.LightTheme.Tabs.Borders.Enabled / userChrome.DarkTheme.Tabs.Borders.Enabled`
`userChrome.LightTheme.Tabs.Shadows.Saturation.Low.Enabled / userChrome.DarkTheme.Tabs.Shadows.Saturation.Low.Enabled`

**Comment by QNetITQ** [comment]
This is not a bug, but a decorative effect. I can remove it if you want. It has no fundamental significance.

**Comment by dreamsynth** [comment]
Yes, if possible. It looks weird when the separators disappear when moving tabs. Thank you!

**Comment by QNetITQ** [comment]
All is ready. This behavior will be removed in the next release.

**Comment by dreamsynth** [comment]
Thank you!

**Comment by dreamsynth** [comment]
I noticed that on floating tabs the separators move strangely when dragging tabs. Now I understand why you were hiding them. However, I came to the decision to just increase the width of the tab background itself, since I couldn't find a way to reduce the gap between tabs in any other way:

```
.tab-background {
    margin-inline: calc(-1 * (var(--tab-min-height) / 1.8)) !important;
}

.tab-content {
    padding-inline: 10px !important;
}
```
